ethers-contract/ethers-contract-abigen/Cargo.toml

```diff
@@ -24,6 +24,10 @@ hex = { version = "0.4.2", default-features = false, features = ["std"] }
 reqwest = { version = "0.11.3", features = ["blocking"] }
 once_cell = { version = "1.8.0", default-features = false }
 cargo_metadata = "0.14.0"
+cfg-if = "1.0.0"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
This doesn't seem to be used anywhere?

Reply:
you're right, we don't actually use this directly, however we need to set this for it to compile standalone:
https://docs.rs/getrandom/0.2.3/getrandom/#indirect-dependencies